src/i18n/jp_JP.json

```diff
@@ -0,0 +1,968 @@
+            "あなたがコメントした提出内容にコメントが付きました",
+          "bounty_expired": "報奨が失効しました",
+          "bounty_changed": "あなたが完了した報奨が変更されました",
+          "bounty_changed_applicant": "応募した報奨が変更されました",
+          "fulfillment_updated_issuer":
+            "フルフィラー、あなたの報奨への提出を更新",
+          "fulfillment_submitted_issuer":
+            "フルフィラー、あなたの報奨への新しいフルフィルメントを提出",
+          "fulfillment_accepted_fulfiller": "報奨作成者があなたの提出を承認しました",
+          "contribution_received":
+            "他のユーザが報奨に寄付しました",
+          "bounty_completed": "報奨には支払いに十分な資金がありません",
+          "application_received": "報奨に新しいフルフィラーが応募しています",
+          "application_accepted_applicant": "報奨作成者があなたの応募を承認しました",
+          "application_rejected_applicant": "報奨作成者があなたの応募を却下しました",
+          "activity": "一般的なアクティビティ（プロファイル更新、報奨の新規作成など）"
+        }
+      },
+      "user": {
+        "photo": {
+          "title": "プロファイルの写真",
+          "label": "画像のアップロードが失敗しました。再度実行して下さい。"
+        },
+        "about": {
+          "title": "あなたについて",
+          "description": "他の人にどのようなことを知らせたいですか？",
+          "notice":
+            "あなたの事がコミュニティに分かるよう、あなた自身の個人的な詳細などを入力して下さい。",
+          "form": {
+            "name": {
+              "label": "名前",
+              "placeholder": "名前を入力…"
+            },
+            "organization": {
+              "label": "組織・団体",
+              "placeholder": "組織・団体を入力…"
+            },
+            "languages": {
+              "label": "言語",
+              "placeholder": "言語を選択…"
+            }
+          }
+        },
+        "skills": {
+          "title": "スキル",
+          "description":
+            "業務や技術に関するスキルは、どのようなものですか？",
+          "notice":
+            "得意とするスキルを入力または選択して下さい。これによって、あなたが一定の型の報奨を完了する技能を有することを本ネットワークの他の人が分かるようになります。",
+          "label": "スキル",
+          "placeholder": "スキルを作成または選択…"
+        },
+        "social": {
+          "title": "ソーシャル",
+          "description":
+            "他に表示を希望するプロファイルや情報はありますか？",
+          "form": {
+            "web": {
+              "label": "個人のウェブサイト"
+            }
+          }
+        },
+        "contact": {
+          "title": "連絡先",
+          "description":
+            "報奨やフルフィルメントに関するステータスの通知をどこで受け取りたいですか？",
+          "form": {
+            "email": {
+              "label": "連絡先のメール",
+              "placeholder": "メールを入力…"
+            },
+            "marketing": {
+              "label":
+                "関連する報奨の示唆やサイトの更新情報の受信も希望します"
+            }
+          },
+          "actions": {
+            "submit": "詳細を送信",
+            "update": "プロファイルを更新"
+          }
+        }
+      }
+    },
+    "submissions": {
+      "title": "送信",
+      "tabs": {
+        "received": "受信しました",
+        "submitted": "送信しました"
+      },
+      "headers": {
+        "title": "報奨のタイトル",
+        "fulfiller": "フルフィラー",
+        "date": "送信日",
+        "status": "ステータス",
+        "payment": "支払い額"
+      },
+      "zero_state": {
+        "title": "あなたの送信は {currentTab} 0 です,
```

Comment:
seems missing a quote

Reply:
@ntkomata I fixed it. Thank you for your pointing it out.

Reply:
hehe no problem